core/types/block.go

```diff
@@ -181,12 +181,13 @@ func init() {
 	BodyRegistry.MustRegister("v1", new(BodyV1))
 }
 
-// Block represents an entire block in the Ethereum blockchain.
+// Block represents an entire block in the Harmony blockchain.
 type Block struct {
-	header           *block.Header
-	uncles           []*block.Header
-	transactions     Transactions
-	incomingReceipts CXReceiptsProofs
+	header              *block.Header
+	uncles              []*block.Header
+	transactions        Transactions
+	stakingTransactions staking.StakingTransactions
```

Comment:
TODO @fxfactorial – add corresponding fields to extblock; use the versioning scheme (extblockV3)